src/backend/tcop/postgres.c

```diff
@@ -5492,6 +5492,7 @@ PostgresMain(int argc, char *argv[],
 
 					SetUserIdAndSecContext(GetOuterUserId(), 0);
 
+					SIMPLE_FAULT_INJECTOR("qe_exec_finished");
```

Comment:
Maybe, Is it more correct way to add all commit, where this line was added in 7X? Or you can add short description to commit to explain difference between your commit and original.

Reply:
So where is the comment (at commit level) about what lines are not related to current backported commit, from what commit they backported and why?

Reply:
added